creative/renderers/display/renderer.js

```diff
@@ -1,6 +1,17 @@
+import { MESSAGE_EVENT } from '../../constants.js';
 import {ERROR_NO_AD} from './constants.js';
 
-export function render({ad, adUrl, width, height, instl}, {mkFrame}, win) {
+export function render({ad, adUrl, width, height, instl}, {mkFrame, sendMessage}, win) {
```

Comment:
You can extract this into some shared file within `creative/`. The 'browserIntervention' event can also be defined once instead of copied as a string, similar to how other events are imported in `creative/constants.js` (the oddity of that file importing just to export is to avoid pulling more of prebid core into the creative).